app/Http/Controllers/EntriesController.php

```diff
@@ -79,16 +79,50 @@ public function add(Request $request)
 
     public function push(Request $request)
     {
+
         $validatedData = $request->validate([
-            'id' => 'required|exists:entries',
+            'id' => 'required|array',
         ]);
-        try {
-            $entry = Entry::findOrFail($validatedData['id']);
-            $entry->push();
-            return redirect('/panel/entries/list')->with('status', 'Entrada Validada Manualmente!');
-        } catch (Exception $e) {
-            return redirect('/panel/entries/list')->with('status', 'Erro ao validar entrada!');
+
+        foreach(request()->get('id') as $id){
+            try {
+                $entry = Entry::findOrFail($id);
+                $entry->update(['used' => 1]);
+
+                $fuel_station = $entry->fuelStation();
+                $fuel_station->update(['has_gasoline' => $entry->has_gasoline,'has_diesel' => $entry->has_diesel, 'has_lpg' => $entry->has_lpg]);
+            } catch (Exception $e) {
+                return redirect('/panel/entries/list')->with('status', 'Erro ao validar entrada!');
+            }
+        }
+
+        $cacheController = new CacheController();
+        $cacheController->updateStations();
```

Comment:
Please, refactor this. Controllers shouldn't be used this way. The logic in the `updateStations()` method should be moved to a service class, which can then be called from the `CacheController` and other controllers, but not this way.